data.sql

```diff
@@ -1,5 +1,7 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name,date_of_birth,escape_attempts, neutered, weight_kg) VALUES ('Agumon','2020-02-03',0,true,10.23);
+INSERT INTO animals (name,date_of_birth,escape_attempts, neutered) VALUES ('Gabumon','2018-11-15',2,true);
```

Comment:
- [ ] you forgot to add the weight_kg for this one so you should do that in order to complete the requirements